crates/test-files/fixtures/features/two_contracts.fe

```diff
@@ -12,7 +12,7 @@ contract Foo {
         return self.other.answer()
     }
 
-    pub fn add(_ x: u256, _ y: u256) -> u256 {
```

Comment:
A lot of these tests in the diff would not need `self` but I don't want to change the meaning of the test. I did add a test that demonstrates that calling pure functions on an external contract works just fine via `ExternalContract::some_func()`